### DISCUSSION TITLE
vo2_max() Criterion?

### DISCUSSION THREAD

**Discussion by Scott-S-Hannah** [fmmattioni/whippr#9] [Ideas]
Hi,

Just a quick idea for discussion :)

You might have already thought about this, but I thought that some VO2max criterion based on literature might be a nice addition. Understandably, this might be difficult given how debated primary and secondary criterion are in the literature. The `vo2_max()` function returns the most common secondary criteria used in the literature (i.e., Heart rate and RER) and I see that you plan to add VO2 plateau detection. It could be as simple as some `ifelse` functions and some logical operators. e.g. output

Primary Criterion 
VO2max plateau = TRUE/FALSE 

Secondary Criterion 
HR = TRUE/FALSE
RER = TRUE/FALSE

Food for thought 👍

**Comment by fmmattioni** [comment]
Hey Scott, that is indeed a really good idea! One thing that I have been struggling with, though, is trying to find a reasonable and robust way to determine a plateau in the VO2 response. Any thoughts on that?
